solr_conf_4_testing/schema.xml

```diff
@@ -235,8 +235,6 @@
     <dynamicField name="*_facet" type="string" stored="true" indexed="true" multiValued="true" />
     <dynamicField name="*_display" type="string" stored="true" indexed="false" multiValued="true"/>
 -->
-    <dynamicField name="*_si" type="string" stored="true" indexed="true" omitNorms="true" />
```

Comment:
:+1: to removing these redundant fields. Before copying these to sul-solr, though, we ought to double check that no data has been indexed into them.